style/client.css

```diff
@@ -494,6 +495,7 @@ p.or:after {
 	border-color: #555555;
 	background: #FFFFFF;
 	background: -webkit-gradient(linear, left top, left bottom, from(#dce4ec), to(#c6d3e0));
+	background: -moz-linear-gradient(top,  #dce4ec, #c6d3e0);
```

Comment:
[It is supported](https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient#Browser_compatibility)